force-app/main/default/classes/VariablesDataTypesOperators.cls

```diff
@@ -226,9 +267,9 @@ public with sharing class VariablesDataTypesOperators {
      */
     public static Double complexOrderOfOperations(Integer a, Integer b, Integer c, Integer d, Integer e) {
         // Add parentheses around the multiplication and subtraction operations so that they are performed before division and addition
-        Double result = a * b - c / (Double) d + e;
+        Double result = (a * b - c ) /  d + e;
```

Comment:
What happens here when you divide by an integer and it would create a result that is not perfectly divisible?

Reply:
I lose the trailing decimals. Do I need the (Double) on both sides of the equation? Ex: Double result = (Double) a * b - c / (Double) d + e;